images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -161,6 +161,75 @@ Hashtable for service arguments
     }
 }
 
+function Install-VsixExtension
+{
+    Param
+    (
+        [String]$Url,
+        [String]$Name
+    )
+
+    $FilePath = "${env:Temp}\$Name"
+    $ReleaseInPath = 'Enterprise'
```

Comment:
Do we really need a hardcoded value?

Reply:
I'm not sure, that part was taken from Install-Wix.ps1, probably we can get rid of it